operator/v1/types_dns.go

```diff
@@ -101,6 +100,24 @@ type DNSSpec struct {
 	// +optional
 	// +kubebuilder:default=Normal
 	LogLevel DNSLogLevel `json:"logLevel,omitempty"`
+
+	// cache describes the caching configuration that applies to all server blocks listed in the Corefile.
+	// This field allows a cluster admin to optionally configure:
+	// * successTTL which is a duration in seconds for which positive responses should be cached.
+	// * denialTTL which is a duration in seconds for which negative responses should be cached.
+	// +optional
+	Cache *DNSCache `json:"cache,omitempty"`
```

Comment:
I think we should also state what would happen by default if the user does not set anything.

Reply:
Added!